<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GanidsMainWindow</class>
 <widget class="QMainWindow" name="GanidsMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GanidsMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button_start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start capture</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Select network interface:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_interfaces">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>5</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>61</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Select CUDA device:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_cuda_devices">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>54</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_stop">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>431</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Statistics</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Packets recevied:</string>
     </property>
    </widget>
    <widget class="QLabel" name="stat_packets_received">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>TCP/IP packets:</string>
     </property>
    </widget>
    <widget class="QLabel" name="stat_tcpip">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Avg. load (Mb/s):</string>
     </property>
    </widget>
    <widget class="QLabel" name="stat_avg_load">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Cur. load (Mb/s):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>481</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Traffic</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_gpu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>39</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable GPU</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="group_cpu_settings">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>CPU settings</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QSpinBox" name="spinBox_3">
     <property name="geometry">
      <rect>
       <x>163</x>
       <y>29</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>35</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of threads:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_gpu_settings">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>GPU settings (buffer flush mode)</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>237</x>
       <y>36</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_time">
     <property name="geometry">
      <rect>
       <x>117</x>
       <y>29</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_timed">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>35</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Timed</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_buffer">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>66</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Buffer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>237</x>
       <y>66</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>bytes</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_buffer_size">
     <property name="geometry">
      <rect>
       <x>117</x>
       <y>59</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1024</number>
     </property>
     <property name="maximum">
      <number>100000000</number>
     </property>
     <property name="singleStep">
      <number>1024</number>
     </property>
     <property name="value">
      <number>65536</number>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="check_save">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>110</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save packets to file:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_save_path">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>102</y>
      <width>491</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_load_path">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>137</y>
      <width>471</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_load">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>145</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load packets from file:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save_path">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>678</x>
      <y>101</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_load_path">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>678</x>
      <y>137</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>511</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>CPU load</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
